### ISSUE TITLE
[Dev Console]: Cannot install mojo on fedora

### ISSUE THREAD

**Issue by arthurbg951** [modularml/mojo#1798]
### Issue description

This was the output from my terminal:

`modular install mojo                                                   ░▒▓ ✔ │ 42s  │ 07:38:35 AM  
modular: /lib64/libtinfo.so.6: no version information available (required by modular)
# Found release for https://packages.modular.com/mojo @ 0.7.0
# Collecting necessary files...
[files    105/105][bytes    133MiB/133MiB @   938KiB/s][██████████████████████████████] 💯%
# Downloads complete, setting configs...
# Configs complete, running post-install hooks...
Collecting find_libpython==0.3.0 (from -r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 1))
  Obtaining dependency information for find_libpython==0.3.0 from https://files.pythonhosted.org/packages/7a/13/4a99ff41ae6a47ccc45675bd312a238b9606b118a9317c79a4277cd2a00a/find_libpython-0.3.0-py3-none-any.whl.metadata
  Downloading find_libpython-0.3.0-py3-none-any.whl.metadata (2.7 kB)
Collecting jupyter_client>=8.3.0 (from -r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for jupyter_client>=8.3.0 from https://files.pythonhosted.org/packages/43/ae/5f4f72980765e2e5e02b260f9c53bcc706cefa7ac9c8d7240225c55788d4/jupyter_client-8.6.0-py3-none-any.whl.metadata
  Downloading jupyter_client-8.6.0-py3-none-any.whl.metadata (8.3 kB)
Collecting ipykernel>=6.25.0 (from -r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for ipykernel>=6.25.0 from https://files.pythonhosted.org/packages/16/9a/0c7b514c73b42cf4ce516ee26c8940a0b23a9754dafaa459a939220240fd/ipykernel-6.29.2-py3-none-any.whl.metadata
  Downloading ipykernel-6.29.2-py3-none-any.whl.metadata (6.0 kB)
Collecting jupyter-core!=5.0.*,>=4.12 (from jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for jupyter-core!=5.0.*,>=4.12 from https://files.pythonhosted.org/packages/86/a1/354cade6907f2fbbd32d89872ec64b62406028e7645ac13acfdb5732829e/jupyter_core-5.7.1-py3-none-any.whl.metadata
  Downloading jupyter_core-5.7.1-py3-none-any.whl.metadata (3.4 kB)
Collecting python-dateutil>=2.8.2 (from jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 6.3 MB/s eta 0:00:00
Collecting pyzmq>=23.0 (from jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for pyzmq>=23.0 from https://files.pythonhosted.org/packages/3a/f1/e296d5a507eac519d1fe1382851b1a4575f690bc2b2d2c8eca2ed7e4bd1f/pyzmq-25.1.2-cp312-cp312-manylinux_2_28_x86_64.whl.metadata
  Downloading pyzmq-25.1.2-cp312-cp312-manylinux_2_28_x86_64.whl.metadata (4.9 kB)
Collecting tornado>=6.2 (from jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for tornado>=6.2 from https://files.pythonhosted.org/packages/9f/12/11d0a757bb67278d3380d41955ae98527d5ad18330b2edbdc8de222b569b/tornado-6.4-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading tornado-6.4-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.5 kB)
Collecting traitlets>=5.3 (from jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for traitlets>=5.3 from https://files.pythonhosted.org/packages/45/34/5dc77fdc7bb4bd198317eea5679edf9cc0a186438b5b19dbb9062fb0f4d5/traitlets-5.14.1-py3-none-any.whl.metadata
  Downloading traitlets-5.14.1-py3-none-any.whl.metadata (10 kB)
Collecting comm>=0.1.1 (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for comm>=0.1.1 from https://files.pythonhosted.org/packages/6e/c1/e7335bd49aa3fa3bd453e34a4580b0076804f219897ad76d4d5aa4d8f22f/comm-0.2.1-py3-none-any.whl.metadata
  Downloading comm-0.2.1-py3-none-any.whl.metadata (3.7 kB)
Collecting debugpy>=1.6.5 (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for debugpy>=1.6.5 from https://files.pythonhosted.org/packages/3c/7a/e400d9f5ed6b4b5ec9f350b4d7974e986ca2d455221e567e648edfad898e/debugpy-1.8.1-cp312-cp312-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading debugpy-1.8.1-cp312-cp312-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.1 kB)
Collecting ipython>=7.23.1 (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for ipython>=7.23.1 from https://files.pythonhosted.org/packages/11/3e/956d40991c152010d71b38b1a9763edc8fe5944d360e1c256fae60cb9165/ipython-8.22.1-py3-none-any.whl.metadata
  Downloading ipython-8.22.1-py3-none-any.whl.metadata (4.8 kB)
Collecting matplotlib-inline>=0.1 (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for matplotlib-inline>=0.1 from https://files.pythonhosted.org/packages/f2/51/c34d7a1d528efaae3d8ddb18ef45a41f284eacf9e514523b191b7d0872cc/matplotlib_inline-0.1.6-py3-none-any.whl.metadata
  Downloading matplotlib_inline-0.1.6-py3-none-any.whl.metadata (2.8 kB)
Collecting nest-asyncio (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for nest-asyncio from https://files.pythonhosted.org/packages/a0/c4/c2971a3ba4c6103a3d10c4b0f24f461ddc027f0f09763220cf35ca1401b3/nest_asyncio-1.6.0-py3-none-any.whl.metadata
  Downloading nest_asyncio-1.6.0-py3-none-any.whl.metadata (2.8 kB)
Collecting packaging (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for packaging from https://files.pythonhosted.org/packages/ec/1a/610693ac4ee14fcdf2d9bf3c493370e4f2ef7ae2e19217d7a237ff42367d/packaging-23.2-py3-none-any.whl.metadata
  Downloading packaging-23.2-py3-none-any.whl.metadata (3.2 kB)
Collecting psutil (from ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for psutil from https://files.pythonhosted.org/packages/c5/4f/0e22aaa246f96d6ac87fe5ebb9c5a693fbe8877f537a1022527c47ca43c5/psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (21 kB)
Collecting decorator (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Downloading decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting jedi>=0.16 (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for jedi>=0.16 from https://files.pythonhosted.org/packages/20/9f/bc63f0f0737ad7a60800bfd472a4836661adae21f9c2535f3957b1e54ceb/jedi-0.19.1-py2.py3-none-any.whl.metadata
  Downloading jedi-0.19.1-py2.py3-none-any.whl.metadata (22 kB)
Collecting prompt-toolkit<3.1.0,>=3.0.41 (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for prompt-toolkit<3.1.0,>=3.0.41 from https://files.pythonhosted.org/packages/ee/fd/ca7bf3869e7caa7a037e23078539467b433a4e01eebd93f77180ab927766/prompt_toolkit-3.0.43-py3-none-any.whl.metadata
  Downloading prompt_toolkit-3.0.43-py3-none-any.whl.metadata (6.5 kB)
Collecting pygments>=2.4.0 (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for pygments>=2.4.0 from https://files.pythonhosted.org/packages/97/9c/372fef8377a6e340b1704768d20daaded98bf13282b5327beb2e2fe2c7ef/pygments-2.17.2-py3-none-any.whl.metadata
  Downloading pygments-2.17.2-py3-none-any.whl.metadata (2.6 kB)
Collecting stack-data (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for stack-data from https://files.pythonhosted.org/packages/f1/7b/ce1eafaf1a76852e2ec9b22edecf1daa58175c090266e9f6c64afcd81d91/stack_data-0.6.3-py3-none-any.whl.metadata
  Downloading stack_data-0.6.3-py3-none-any.whl.metadata (18 kB)
Collecting pexpect>4.3 (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for pexpect>4.3 from https://files.pythonhosted.org/packages/9e/c3/059298687310d527a58bb01f3b1965787ee3b40dce76752eda8b44e9a2c5/pexpect-4.9.0-py2.py3-none-any.whl.metadata
  Downloading pexpect-4.9.0-py2.py3-none-any.whl.metadata (2.5 kB)
Collecting platformdirs>=2.5 (from jupyter-core!=5.0.*,>=4.12->jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Obtaining dependency information for platformdirs>=2.5 from https://files.pythonhosted.org/packages/55/72/4898c44ee9ea6f43396fbc23d9bfaf3d06e01b83698bdf2e4c919deceb7c/platformdirs-4.2.0-py3-none-any.whl.metadata
  Downloading platformdirs-4.2.0-py3-none-any.whl.metadata (11 kB)
Collecting six>=1.5 (from python-dateutil>=2.8.2->jupyter_client>=8.3.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2))
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting parso<0.9.0,>=0.8.3 (from jedi>=0.16->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Downloading parso-0.8.3-py2.py3-none-any.whl (100 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100.8/100.8 kB 6.1 MB/s eta 0:00:00
Collecting ptyprocess>=0.5 (from pexpect>4.3->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Downloading ptyprocess-0.7.0-py2.py3-none-any.whl (13 kB)
Collecting wcwidth (from prompt-toolkit<3.1.0,>=3.0.41->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for wcwidth from https://files.pythonhosted.org/packages/fd/84/fd2ba7aafacbad3c4201d395674fc6348826569da3c0937e75505ead3528/wcwidth-0.2.13-py2.py3-none-any.whl.metadata
  Downloading wcwidth-0.2.13-py2.py3-none-any.whl.metadata (14 kB)
Collecting executing>=1.2.0 (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for executing>=1.2.0 from https://files.pythonhosted.org/packages/80/03/6ea8b1b2a5ab40a7a60dc464d3daa7aa546e0a74d74a9f8ff551ea7905db/executing-2.0.1-py2.py3-none-any.whl.metadata
  Downloading executing-2.0.1-py2.py3-none-any.whl.metadata (9.0 kB)
Collecting asttokens>=2.1.0 (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Obtaining dependency information for asttokens>=2.1.0 from https://files.pythonhosted.org/packages/45/86/4736ac618d82a20d87d2f92ae19441ebc7ac9e7a581d7e58bbe79233b24a/asttokens-2.4.1-py2.py3-none-any.whl.metadata
  Downloading asttokens-2.4.1-py2.py3-none-any.whl.metadata (5.2 kB)
Collecting pure-eval (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3))
  Downloading pure_eval-0.2.2-py3-none-any.whl (11 kB)
Downloading find_libpython-0.3.0-py3-none-any.whl (8.5 kB)
Downloading jupyter_client-8.6.0-py3-none-any.whl (105 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 105.9/105.9 kB 4.7 MB/s eta 0:00:00
Downloading ipykernel-6.29.2-py3-none-any.whl (116 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 116.1/116.1 kB 5.8 MB/s eta 0:00:00
Downloading comm-0.2.1-py3-none-any.whl (7.2 kB)
Downloading debugpy-1.8.1-cp312-cp312-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.4 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.4/1.4 MB 4.0 MB/s eta 0:00:00
Downloading ipython-8.22.1-py3-none-any.whl (811 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 812.0/812.0 kB 2.8 MB/s eta 0:00:00
Downloading jupyter_core-5.7.1-py3-none-any.whl (28 kB)
Downloading matplotlib_inline-0.1.6-py3-none-any.whl (9.4 kB)
Downloading pyzmq-25.1.2-cp312-cp312-manylinux_2_28_x86_64.whl (1.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 1.9 MB/s eta 0:00:00
Downloading tornado-6.4-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (435 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 435.4/435.4 kB 2.9 MB/s eta 0:00:00
Downloading traitlets-5.14.1-py3-none-any.whl (85 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 85.4/85.4 kB 2.1 MB/s eta 0:00:00
Downloading nest_asyncio-1.6.0-py3-none-any.whl (5.2 kB)
Downloading packaging-23.2-py3-none-any.whl (53 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.0/53.0 kB 1.1 MB/s eta 0:00:00
Downloading psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (288 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 288.2/288.2 kB 6.8 MB/s eta 0:00:00
Downloading jedi-0.19.1-py2.py3-none-any.whl (1.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 9.8 MB/s eta 0:00:00
Downloading pexpect-4.9.0-py2.py3-none-any.whl (63 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.8/63.8 kB 1.7 MB/s eta 0:00:00
Downloading platformdirs-4.2.0-py3-none-any.whl (17 kB)
Downloading prompt_toolkit-3.0.43-py3-none-any.whl (386 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 386.1/386.1 kB 9.4 MB/s eta 0:00:00
Downloading pygments-2.17.2-py3-none-any.whl (1.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 9.3 MB/s eta 0:00:00
Downloading stack_data-0.6.3-py3-none-any.whl (24 kB)
Downloading asttokens-2.4.1-py2.py3-none-any.whl (27 kB)
Downloading executing-2.0.1-py2.py3-none-any.whl (24 kB)
Downloading wcwidth-0.2.13-py2.py3-none-any.whl (34 kB)
Installing collected packages: wcwidth, pure-eval, ptyprocess, find_libpython, traitlets, tornado, six, pyzmq, pygments, psutil, prompt-toolkit, platformdirs, pexpect, parso, packaging, nest-asyncio, executing, decorator, debugpy, python-dateutil, matplotlib-inline, jupyter-core, jedi, comm, asttokens, stack-data, jupyter_client, ipython, ipykernel
Successfully installed asttokens-2.4.1 comm-0.2.1 debugpy-1.8.1 decorator-5.1.1 executing-2.0.1 find_libpython-0.3.0 ipykernel-6.29.2 ipython-8.22.1 jedi-0.19.1 jupyter-core-5.7.1 jupyter_client-8.6.0 matplotlib-inline-0.1.6 nest-asyncio-1.6.0 packaging-23.2 parso-0.8.3 pexpect-4.9.0 platformdirs-4.2.0 prompt-toolkit-3.0.43 psutil-5.9.8 ptyprocess-0.7.0 pure-eval-0.2.2 pygments-2.17.2 python-dateutil-2.8.2 pyzmq-25.1.2 six-1.16.0 stack-data-0.6.3 tornado-6.4 traitlets-5.14.1 wcwidth-0.2.13

[notice] A new release of pip is available: 23.2.1 -> 24.0
[notice] To update, run: pip install --upgrade pip
[mojo][error] Mojo SDK post-install test "`mojo repl`" failed:
SubprocessError: Command '/home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo repl --one-line-before-file settings set target.load-cwd-lldbinit false --one-line-before-file settings set plugin.process.gdb-remote.packet-timeout 60' returned non-zero exit status 127.
  stderr:
    /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo: /lib64/libtinfo.so.6: no version information available (required by /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo)
    /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo-lldb: error while loading shared libraries: libedit.so.2: cannot open shared object file: No such file or directory
  File "/home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py", line 295, in test_mojo_repl
    raise SubprocessError(repl.returncode, " ".join(args), out, err)
[mojo][error] Some components of the Mojo SDK may have been installed successfully, but others may not work as expected.
[mojo][error] Please submit an issue to https://github.com/modularml/mojo and include the full output of the command you just ran.
modular: error: failed to run script
==========================
Failure Information:
	- Script: `PATH=/home/arthur/miniconda3/condabin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/home/arthur/.dotnet/tools:/home/arthur/Documents/gmsh-4.12.2-Linux64/bin:/home/arthur/.local/bin HOME=/home/arthur PYTHONPATH=/home/arthur/.modular/pkg/packages.modular.com_mojo/venv/lib/home/arthur/.modular/pkg/packages.modular.com_mojo/venv/bin/python /home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py --modular-home /home/arthur/.modular --package-path /home/arthur/.modular/pkg/packages.modular.com_mojo`
	- Result: 1
	- Stderr:
[mojo][error] Mojo SDK post-install test "`mojo repl`" failed:
SubprocessError: Command '/home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo repl --one-line-before-file settings set target.load-cwd-lldbinit false --one-line-before-file settings set plugin.process.gdb-remote.packet-timeout 60' returned non-zero exit status 127.
  stderr:
    /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo: /lib64/libtinfo.so.6: no version information available (required by /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo)
    /home/arthur/.modular/pkg/packages.modular.com_mojo/bin/mojo-lldb: error while loading shared libraries: libedit.so.2: cannot open shared object file: No such file or directory
  File "/home/arthur/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py", line 295, in test_mojo_repl
    raise SubprocessError(repl.returncode, " ".join(args), out, err)
[mojo][error] Some components of the Mojo SDK may have been installed successfully, but others may not work as expected.
[mojo][error] Please submit an issue to https://github.com/modularml/mojo and include the full output of the command you just ran.`

### Steps to reproduce

Step1:
curl https://get.modular.com | sh - && \
modular auth mut_43d6d2298c954cb182fe08558c69a77f

Step2:
modular install mojo

**Comment by Sharktheone** [comment]
Duplicate of #855 

TL;DR use [my script](https://github.com/Sharktheone/arch-mojo) to install mojo on arch or fedora

**Comment by arthurbg951** [comment]
This script must be used with my system python interpreter or with a conda environment?

**Comment by Sharktheone** [comment]
> This [script](https://github.com/Sharktheone/arch-mojo) must be used with my system python interpreter or with a conda environment? @Sharktheone

It doesn't matter

**Comment by arthurbg951** [comment]
It works fine. But I suggest you to comment that this script only works with a python 3.10 +. Thanks!

**Comment by Sharktheone** [comment]
> It works fine. But I suggest you to comment that this script only works with a python 3.10 +. Thanks!

Thanks for the feedback!
